src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -134,7 +177,8 @@ object Service extends SimpleRoutingApp with DataHubCatalog with Instrumented wi
       } ~
       pingPong ~
       pathPrefix("layers"){registerLayerRoute} ~
-      pathPrefix("breaks"){registerColorBreaksRoute}
+      pathPrefix("intbreaks"){registerIntColorBreaksRoute} ~
+      pathPrefix("floatbreaks"){registerFloatColorBreaksRoute}
```

Comment:
Option 2 looks best to me, and would require the smallest amount of downstream code changes.